JSONSchemaPrettyPrinter/TWTJSONSchemaPrettyPrinter.h

```diff
@@ -29,6 +29,6 @@
 
 @interface TWTJSONSchemaPrettyPrinter : NSObject <TWTJSONSchemaASTProcessor>
 
-- (NSDictionary *)objectFromSchema:(TWTJSONSchemaTopLevelASTNode *)topLevelNode;
```

Comment:
More consistently differentiate between "schema" (to mean NSDictionary) and "node"